### ISSUE TITLE
Number of hospitalized patients is a floating point number

### ISSUE THREAD

**Issue by eguidotti** [M3IT/COVID-19_Data#9]
Hi Matt, 
hope you are doing well! 

@MinerMax555 noticed that for some dates (example: 2021-01-01) the number of hospitalized people in Australia is a floating point number (20.44 on 2021-01-01). 

- https://raw.githubusercontent.com/M3IT/COVID-19_Data/master/Data/COVID19_Data_Hub.csv

Do you know what this floating point number means? Is it a data mistake or am I allowed to ruond it to the nearest integer?

Please see also the corresponding issue on the data hub repo:

- https://github.com/covid19datahub/COVID19/issues/193

Many thanks!

**Comment by M3IT** [comment]
Thanks for bringing this to my attention Emanuele.

I've looked into it and the reasons are:

For hospitalisations, in some states the numbers provided are:

- # being treated (Includes hospital at home)
- % of those who are hospitalised

So the precise number of hospitalisations isn't known, but is calculated from these 2 numbers.

For vaccines, second doses are provided by some states as % of population, so again the number is calculated.

I've fixed this by rounding all counts to the nearest integer and then coercing to integer so they stay that way downstream (and added a check). This fix is in tonight's update.
